cardano-node/src/Cardano/Tracing/Peer.hs

```diff
@@ -57,7 +58,14 @@ ppPeer (Peer cid _af status inflight) =
   Text.pack $ printf "%-15s %-8s %s" (ppCid cid) (ppStatus status) (ppInFlight inflight)
 
 ppCid :: RemoteConnectionId -> String
-ppCid = takeWhile (/= ':') . show . remoteAddress
+ppCid cid =
+  case r of
+    SockAddrInet{} -> takeWhile (/= ':') s
+    SockAddrInet6{} -> takeWhile (/= ']') s ++ [']']
+    _ -> s
+  where
+    r = remoteAddress cid
+    s = show r
```

Comment:
The port number is part of the connection id. Two TCP connections from the same IP address but different remote ports are treated as different peers by ourboros-network.

Reply:
oh, the previous implementation was incorrect for IPv4, too?

Reply:
Yes.